runtime/jcl/common/mgmtthread.c

```diff
@@ -231,7 +231,13 @@ Java_com_ibm_java_lang_management_internal_ThreadMXBeanImpl_getAllThreadIdsImpl(
 	threadCount = 0;
 	do {
 		{
-			if ((currentThread->threadObject != NULL) && (J9VMJAVALANGTHREAD_THREADREF((J9VMThread *)env,currentThread->threadObject) != NULL)) {
+			if ((NULL != currentThread->threadObject)
+#if defined(J9VM_OPT_OJDK_THREAD_SUPPORT)
+				&& (J9VMJAVALANGTHREAD_EETOP((J9VMThread *)env,currentThread->threadObject) != NULL)
```

Comment:
Constant on the left here and in the else.

Reply:
The change to this file is not required now, so skip format only change.